server/controllers/Events/resolver.ts

```diff
@@ -47,6 +47,7 @@ export class EventResolver {
 
     event.name = data.name ?? event.name;
     event.description = data.description ?? event.description;
+    event.url = data.url ?? event.url;
```

Comment:
I would recommend adding some validation before here to ensure that the url being added is indeed a URL, but this is more of a nice-to-have, and not a blocker.

Reply:
Should be easy by passing it to `new URL()` though (in a try-catch block).